### PR TITLE
System.Text.Json CVE fix

### PR DESCRIPTION
**Pull request by keegan-caruso**
[7x-deps.txt](https://github.com/user-attachments/files/16254830/7x-deps.txt)

### CODE REVIEW

**Review by jmprieur — APPROVED** [review]
LGTM
Thanks @keegan-caruso

File: src/Microsoft.IdentityModel.JsonWebTokens/Microsoft.IdentityModel.JsonWebTokens.csproj [hunk 1]
```diff
@@ -27,11 +27,6 @@
     <ProjectReference Include="..\Microsoft.IdentityModel.Tokens\Microsoft.IdentityModel.Tokens.csproj" />
   </ItemGroup>
 
-  <ItemGroup Condition="'$(TargetFrameworkIdentifier)' != '.NETCoreApp'">
```
Comment (pmaytak) [inline src/Microsoft.IdentityModel.JsonWebTokens/Microsoft.IdentityModel.JsonWebTokens.csproj:30]: Why is this different from other csproj changes? Why don't keep a reference to STJ for !.NeTCoreApp?

Comment (keegan-caruso) [reply src/Microsoft.IdentityModel.JsonWebTokens/Microsoft.IdentityModel.JsonWebTokens.csproj:30]: These should all be transitive deps already from src/Microsoft.IdentityModel.Tokens/Microsoft.IdentityModel.Tokens.csproj

**Review by pmaytak — APPROVED** [review]

**Review by pmaytak — APPROVED** [review]

**Review by jennyf19 — APPROVED** [review]
:shipit:

**Pull request merged**